code/__DEFINES/is_helpers.dm

```diff
@@ -89,6 +89,8 @@ GLOBAL_LIST_INIT(turfs_openspace, typecacheof(list(
 //More carbon mobs
 #define isalien(A) (istype(A, /mob/living/carbon/alien))
 
+#define isalienhumanoid(A) (istype(A, /mob/living/carbon/alien/humanoid))
+
 #define islarva(A) (istype(A, /mob/living/carbon/alien/larva))
 
 #define isalienadult(A) (istype(A, /mob/living/carbon/alien/humanoid) || istype(A, /mob/living/simple_animal/hostile/alien))
```

Comment:
remove this please and replace with isalienhumanoid. it is a dumb fucking function that will lets you transform simplemob aliens to carbon praetorians, it makes no sense

Reply:
I feel like you'd be better off just making the change in that one place where transformation happens, rather then everywhere

Reply:
to quote myself, it is used in:
emote code where it doesnt work, because the emote is only given to carbon aliens
queen code which evolves simplemobs to carbon praetorians
slime code where it doesnt run because its under a for loop for carbons
firedoor tooltip code where it lies to you because you cant open firelocks with lmb as a simplemob
and photocopier code, where its useless because it already checks separately for alienadult AND simplemob alien
its pointless and you should delete it

Reply:
I didn't get all of them, because some seemed at least possibly useful. Rest got cleaned tho

Reply:
WHat do you mean some you only kept 1 use of it and that use already literally included an || statement to check for aliens

Reply:
I kept two uses, one in emotes and one in ass code
I figured that was enough, idk maybe it's too few to deserve life